projects/storefrontlib/src/cms-components/navigation/navigation/navigation-ui.component.ts

```diff
@@ -47,18 +49,28 @@ export class NavigationUIComponent implements OnDestroy {
 
   private subscription: Subscription;
 
-  constructor(private router: Router, private renderer: Renderer2) {
+  @HostListener('window:resize')
+  onResize() {
```

Comment:
shouldn't we add some kind of debouncer here? performance might be affected by many resizes

Reply:
that's nice idea